lib/f_core/os/c_task.cpp

```diff
@@ -20,13 +20,16 @@ static void taskEntryWrapper(void* taskObj, void*, void*) {
     while (true) {
         task->Run();
 #if defined(CONFIG_ARCH_POSIX)
-        k_cpu_idle(); // Refer to Zephyr's POSIX arch limitations documentation
+        // Refer to Zephyr's POSIX arch limitations documentation
+        // https://docs.zephyrproject.org/latest/boards/native/native_sim/doc/index.html
+        k_cpu_idle();
 #endif
     }
 }
 
 CTask::CTask(const char* name, int priority, int stackSize, int sleepTimeMs) : name(name),
-    priority(priority), stackSize(stackSize), sleepTimeMs(sleepTimeMs) {
+                                                                               priority(priority), stackSize(stackSize),
```

Comment:
this formatting looks really bizarre in the diff lol

Reply:
x)